test/ctia/schemas/graphql/helpers_test.clj

```diff
@@ -1,26 +1,97 @@
 (ns ctia.schemas.graphql.helpers-test
   (:require [ctia.schemas.graphql.helpers :as sut]
-            [clojure.test :as t :refer [deftest is testing]]))
+            [clj-momo.test-helpers.core :as mth]
+            [clojure.test :as t :refer [deftest is testing use-fixtures]])
+  (:import [java.util.concurrent CountDownLatch TimeUnit]))
+
+(use-fixtures :once mth/fixture-schema-validation)
+
+;; TODO when global graphql schema is stubbable, add parallel tests to new-object-test etc
 
 (deftest new-object-test
   (testing "The same object is not created twice"
-    (is (= (sut/new-object "Object"
+    (let [registry (sut/create-named-type-registry)
+          new-object-fn #(sut/new-object
+                           ;; graphql schema is currently global, so rerunning test will
+                           ;; use cached type
+                           "NewObjectTestObject"
                            "Description 1"
                            []
-                           {})
-           (sut/new-object "Object"
-                           "Description 2"
-                           []
-                           {})))))
+                           {}
+                           registry)]
+      (is (= (new-object-fn)
```

Comment:
Given the `testing "The same object is not created twice"`, you might want to favor `identical?` over `=`

Reply:
Agreed.